hs_app_netCDF/templates/pages/netcdfresource.html

```diff
@@ -178,6 +178,19 @@ <h3 style="margin-top:40px"> NetCDF Header Info</h3>
             {% endfor %}
     {% endif %}
 
+
+        {# Add netcdf opendap button  #}
+    {% if opendap_link %}
+        {% for f in cm.files.all %}
+            {% if f.resource_file.name|slice:"-3:" == '.nc'%}
+                <td><a style="margin-left:15px" id="btn-netcdf-opendap" type="button" class="btn btn-success" href='{{opendap_link }}'
+                       title='When changing this resource to or from public or private,please allow 2 hours for the OpenDAP service to refresh on this page' >
```

Comment:
@gantian127 @rayi113 Do we need to put this text here now that we implemented to refresh THREDDS data server at 2 minutes interval? At least we need to replace "2 hours" with "2 minutes," right?